x-pack/plugins/cases/public/components/case_view/case_view_page.tsx

```diff
@@ -36,7 +29,7 @@ import { useOnUpdateField } from './use_on_update_field';
 // This hardcoded constant is left here intentionally
 // as a way to hide a wip functionality
 // that will be merge in the 8.3 release.
-const ENABLE_ALERTS_TAB = false;
+const ENABLE_ALERTS_TAB = true;
```

Comment:
Should we remove the constant?

Reply:
I think let's keep it here and `false` for this PR and I'll remove the constant in a follow up PR with the full integration.